### PR TITLE
Bump OWL API version to 4.2.5

### PR DESCRIPTION
**Pull request by Brian-McG**
Likely the last PR for a while unless I identify something important to modify.  
##### Changes:
- The OWL API version includes the suggested OWL 2 Full changes https://github.com/owlcs/owlapi/issues/508
- Re-ordered checkboxs to be more logical
- Update error messages
- Code base cleanup

### CODE REVIEW

File: src/main/java/com/oe/UI.java [hunk 1]
```diff
@@ -137,7 +133,7 @@ public void setSelected(boolean b) {
             //		gbc.weighty = 0; gbc.weightx = 0;
 
             //Remove the checkBoxes MouseListener so that it doesnt seem as if you can click it
-            MouseListener[] ml = (MouseListener[]) checkBoxes[i].getListeners(MouseListener.class);
+            MouseListener[] ml = checkBoxes[i].getListeners(MouseListener.class);
```
Comment (Brian-McG) [inline src/main/java/com/oe/UI.java]: Intelij indicated that the ```(MouseListener[])``` was redundant.

**Comment by Brian-McG** [comment]
@muhummadPatel any feedback?

**Comment by muhummadPatel** [comment]
Sorry about that. Tested it out. Looks good. Go ahead and merge this 👍 😄 Also, maybe put out a new release.

**Comment by Brian-McG** [comment]
@muhummadPatel Could you update description text of repo to ```OWL 2 and OWL 1 ontology classifier```

**Comment by muhummadPatel** [comment]
@Brian-McG, repo description changed as requested.

**Pull request merged**